change/react-native-windows-2019-09-18-10-27-56-HEAD.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
```

Comment:
FYI, by specifying none here, you are telling the publish build not to both producing a new version of react-native-windows for this change. -- So a new build will not be published until someone else makes a change.

This means office wont be able to pickup this change until then.

Reply:
> FYI, by specifying none here, you are telling the publish build not to both producing a new version of react-native-windows for this change. -- So a new build will not be published until someone else makes a change.
> This means office wont be able to pickup this change until then.

Ah .. i didn't know that. Thanks!
Anyways, the V8 integration is under a conditional compilation flag. We don't consume the published bits of react-native-win32.dll, instead, we manually build locally enabling the macro. Hence, it should be fine for this PR.